static/src/stylesheets/components/sass-mq/_mq.scss

```diff
@@ -33,6 +33,7 @@ $mq-responsive: true !default;
 /// @link https://github.com/sass-mq/sass-mq#seeing-the-currently-active-breakpoint Full documentation and examples
 $mq-breakpoints: (
     mobile:  320px,
+    phablet: 620px,
```

Comment:
You've changed a bower module here - we already have phablet set as a breakpoint width in `_vars.scss`, so you shouldn't need this, any reason why you added it?